.ci/travis/travis.sh

```diff
@@ -69,6 +69,7 @@ no-error-test-sbe)
   mvn clean install -Pno-validations
   git clone https://github.com/real-logic/simple-binary-encoding.git
   cd simple-binary-encoding
+  git checkout $(git describe --tags `git rev-list --tags --max-count=1`)
```

Comment:
What does this do exactly?
Is there a reason we don't checkout a tag like [1.7.0](https://github.com/real-logic/simple-binary-encoding/tree/1.7.0)?